scripts/check-form.js

```diff
@@ -72,8 +72,6 @@ const request = async (url, method = 'GET', postData) => {
 	});
 };
 
-const initial = '11db17a21add028b8d12930a8b047af1df2d3194'; // git rev-list --max-parents=0 HEAD
-
```

Comment:
this is left intentionally, despite being unused, because it's not trivial to remember how to find it later. Please restore it.